templates/definition/meter/huawei-sun2000-dongle-powersensor.yaml

```diff
@@ -4,6 +4,14 @@ products:
     description:
       generic: SUN2000 with SDongle & Power Sensor
 capabilities: ["battery-control"]
+requirements:
+  description:
+    de: |
+      Es kann notwendig sein, dass der Installateur mit Hilfe eines "Errichterzugangs" zunächst "Modbus/TCP" in den Kommunikationseinstellungen des Wechselrichters freischaltet. 
```

Comment:
Ich fände eine knappe Formulierung besser: 

    Erfordert ModbusTCP. Siehe ...

Reply:
Ich bin auch immer für knappe Formulierungen, aber ich hätte mit "Erfordert MobusTCP" nichts anfangen können :D Dann wäre ich nicht in der Lage gewesen meinem Installateur zu sagen, was ich haben will.... 
Und der Link könnte potentiell in Zukunft mal kaputt gehen, wenn Huawei die Doku umzieht...

Reply:
Erfordert ModbusTCP. Freischaltung über den "Errichterzugang". Siehe ...

Reply:
passe ich an :)